.github/workflows/windows_nightly.yml

```diff
@@ -20,36 +20,90 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Nightly test Windows py${{ matrix.python-version }}
+        env:
+          BAZEL_VC: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/"
+          BAZEL_PATH: /d/a/addons/addons/bazel-${BAZEL_VERSION}-windows-x86_64.exe
         shell: bash
         run: |
           python -m pip install wheel setuptools
           curl -sSOL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-windows-x86_64.exe
-          export BAZEL_VC=/c/Program\ Files\ \(x86\)/Microsoft\ Visual\ Studio/2019/Enterprise/VC/
           export BAZEL_PATH=/d/a/addons/addons/bazel-${BAZEL_VERSION}-windows-x86_64.exe
```

Comment:
Good catch! I've removed it from ENV since it's built on a global env (BAZEL_VERSION) which AFAIK we can't reuse in the yaml until the shell is launched.

Reply:
Got it. Thanks for clarification!